### ISSUE TITLE
[BUG] Order has been filled on RadarRelay but remains on hummingbot 

### ISSUE THREAD

**Issue by rennel-tabing** [CoinAlpha/hummingbot#1152]
**Describe the bug**
// A clear and concise description of what the bug is.
Running on Pure MM. An incident was reported where in orders were filled on the actual Radar Relay account but remains active on the Hummingbot side

**Steps To Reproduce**

Here are the steps to reproduce the issue (see attachments in the section below):
1. Set up your configuration file with these parameters: ...
[PureMM-strategy file.txt](https://github.com/CoinAlpha/hummingbot/files/3912026/PureMM-strategy.file.txt)
...

**Screenshots**
// If applicable, add screenshots to help explain your problem.

Here is the Hummingbot CLI showing that it has been filled but the order remains active.
![image](https://user-images.githubusercontent.com/53868268/69973555-b1bf4e80-14f1-11ea-9464-c5669bcc8f08.png)

Here was the trade on the exchange showing that the trade was filled.
![image](https://user-images.githubusercontent.com/53868268/69973640-d7e4ee80-14f1-11ea-8950-03525a8215fb.png)

Since the order was not cleared and remains active on the Hummingbot side, no new order was placed on the actual exchange site
![image](https://user-images.githubusercontent.com/53868268/69973784-11b5f500-14f2-11ea-895c-88c0d4d09839.png)

**Release version**
// Include your bot's version number (Can be found at the upper left corner of your CLI).
0.19.1

**Attachments**
// Please attach your config file and log file to help us debug. It would be difficult for us to help you without those! 

Config file:
[PureMM-strategy file.txt](https://github.com/CoinAlpha/hummingbot/files/3912071/PureMM-strategy.file.txt)

Error Logs
[Log file.txt](https://github.com/CoinAlpha/hummingbot/files/3912074/Log.file.txt)

*WARNING: Do NOT publish any exchange API keys or your wallet's private key. Whoever has access to them may steal your assets!*

**Optional: your discord username:** fernandobender#8638
**Optional: your ETH address:** 0x78F932824880505690B9e14Cb0097aA3E3B7Fe00

**Comment by chowryan** [comment]
was your order stuck on hummingbot indefinitely? or did it clear eventually?

**Comment by rennel-tabing** [comment]
@chowryan order was stuck indefinitely on the hummingbot side.

**Comment by BUDDHA-TONY** [comment]
Tested twice and cannot reproduce the behavior on my end.After the order has been filled it wasn't stuck both on the exchange and in hummingbot.

![gitRR](https://user-images.githubusercontent.com/55869559/70282106-dc621d80-17f7-11ea-9999-2345d4f6ac12.png)

                "commit": "cf857f346ccbd2ecf301b39a1484c60003d00bdb",
                "date": "2019-12-05T02:07:21Z",
                "maintainer": "CoinAlpha, Inc. <dev@coinalpha.com>"

**Comment by znerol19619** [comment]
I've been running a bot for 20 hrs using the same pair (WETH-DAI) and configuration and haven't encounter this behavior. 

a sell order has been filled, there are orders created and no orders were stuck in hummingbot and radar relay that could have prevent the bot from creating an order. 

**Screenshots**

![radarwebsite_order](https://user-images.githubusercontent.com/27669951/70314790-275d4e80-1853-11ea-8a47-3cc60dce7fbe.png)

**Version**
```
          "branch": "development"
          "commit": "cf857f346ccbd2ecf301b39a1484c60003d00bdb",
          "date": "2019-12-05T02:07:21Z",

```

**Comment by CrimsonJacket** [comment]
Initial investigation seems to suggest that the formatting of the price/amount in fix #1050 might be the reason for this bug.

However, on closer inspection, the `_update_order_status` function in `Market` class should handle and loose orders; stop tracking any orders that is no longer active on the exchange. Several tests have been done and none was able to replicate this bug.

Issue closed 👨‍⚖

**Comment by rennel-tabing** [comment]
as per user he did not noticed any similar incident so far.
